services/snapshotter/service.go

```diff
@@ -182,7 +188,15 @@ func (s *Service) updateMetaStore(conn net.Conn, bits []byte, backupDBName, rest
 		return fmt.Errorf("failed to decode meta: %s", err)
 	}
 
-	data := s.MetaClient.Data()
```

Comment:
I testing this in 1.9.3 Enterprise - it `panic`s here because we in 1.9.3 we do an unchecked type assertion. A recent OSS PR fixed the unchecked type assertion, which broke the Enterprise build. This PR does the right thing and checks the type assertion.